pr_agent/tools/pr_code_suggestions.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, pr_url: str, cli_mode=False):
                                           settings.pr_code_suggestions_prompt.user)
 
     async def suggest(self):
-        assert type(self.git_provider) == GithubProvider, "Only Github is supported for now"
+        assert type(self.git_provider) != BitbucketProvider, "Bitbucket is not supported for now"
```

Comment:
In the `suggest` method, it would be better to handle the case where the git provider is not supported. This can be done by raising a specific exception instead of using an assertion. [medium]